src/cli-client/internals/cli/cli_main.go

```diff
@@ -0,0 +1,59 @@
+package cli
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/canonical/go-flags"
+	"github.com/canonical/oci-factory/cli-client/internals/logger"
+)
+
+type CmdRelease struct {
+	Revision int `long:"revision" description:"The revision number of the image release"`
+}
+
+var opts struct {
+	// TODO: Leave `release` here for now. Should be moved into a `cmd_release.go` in phase 2.
```

Comment:
Then I'd remove/comment it. Otherwise people will try to use it (because it's described in the usage info) and fail.

Reply:
Good catch, fixed.